src/bin/tikv-importer.rs

```diff
@@ -3,10 +3,7 @@
 #![feature(slice_patterns)]
 #![feature(proc_macro_hygiene)]
 
-#[macro_use(
-    slog_crit,
-    slog_info,
-)]
+#[macro_use(slog_crit, slog_info)]
 extern crate slog;
 #[macro_use]
 extern crate slog_global;
```

Comment:
do we need `extern crate` in rust 2018 edition?

Reply:
We don't. It's needed just to import all macros everywhere via `#[macro_rules]`. Rewriting these isn't high priority.